kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -1108,14 +1111,12 @@ define([
 
         function stopJobUpdates() {
             if (listeningForJob) {
```

Comment:
how come it no longer need to emit request-job-completion?

Reply:
The `request-job-completion` command will tell the lookup thread to stop looking up the job status. If there's any other widgets waiting on job information (like the status widget), it'll never see a change. Instead, I just have the log viewer stop listening for updates.

Reply:
the status widget share the same thread?